.travis.yml

```diff
@@ -11,20 +11,20 @@ branches:
 language: generic
 
 env:
-  - NPM_TAG=latest IMAGE_NAME=theia-full
-  - NPM_TAG=next IMAGE_NAME=theia-full
-  - NPM_TAG=latest IMAGE_NAME=theia
-  - NPM_TAG=next IMAGE_NAME=theia
-  - NPM_TAG=latest IMAGE_NAME=theia-java
-  - NPM_TAG=next IMAGE_NAME=theia-java
-  - NPM_TAG=latest IMAGE_NAME=theia-go
-  - NPM_TAG=next IMAGE_NAME=theia-go
-  - NPM_TAG=latest IMAGE_NAME=yangster
-  - NPM_TAG=next IMAGE_NAME=yangster
-  - NPM_TAG=latest IMAGE_NAME=theia-python
-  - NPM_TAG=next IMAGE_NAME=theia-python
-  - NPM_TAG=latest IMAGE_NAME=theia-swift
-  - NPM_TAG=next IMAGE_NAME=theia-swift
+  - NPM_TAG=latest IMAGE_NAME=theia-full NODE_VERSION=8
+  - NPM_TAG=next IMAGE_NAME=theia-full NODE_VERSION=10
```

Comment:
update: it's actually not working for me locally, to use node version "10" for next; we need to pin it to something specific. e.g. using `10.15.3` works, resolving download link for archive to  `https://nodejs.org/dist/v10.15.3/node-v10.15.3-linux-x64.tar.xz`